readme.ipynb

```diff
@@ -49,8 +49,9 @@
     "def func():\n",
```

Comment:
<p>This for loop doesn't look great!</p><p>Also I don't like the variable name at LINE 2!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/epalese/reviewnb-test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>